src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/AddPersonnelSideSheet/AddPersonnelForm/components/AddPersonnelFormLine.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import * as styles from '../../styles.less';
+import Personnel from '../../../../../../../../../models/Personnel';
+import SelectionCell from '../../../components/SelectionCell';
+import AddPersonnelFormTextInput from './AddPersonnelFormTextInput';
+import { SkeletonBar } from '@equinor/fusion-components';
+import AddPersonnelFormDisciplinesDropDown from './AddPersonnelFormDisciplinesDropDown';
+import useForm from '../../../../../../../../../hooks/useForm';
+import { BasePosition } from '@equinor/fusion';
+import { PopOverMenu } from './AddPersonnelFormLinePopOverMenu';
+import PersonnelLine from '../../models/PersonnelLine';
+
+type AddPersonnelFormLineProps = {
+    person: PersonnelLine;
+    setPersonState: (person: PersonnelLine) => void;
+    saveInProgress: boolean;
+    isFetchingBasePositions: Boolean;
+    basePositions: BasePosition[];
+    deletePerson: (person: PersonnelLine) => void;
+};
+
+export const AddPersonnelFormLine: React.FC<AddPersonnelFormLineProps> = ({
```

Comment:
Would `AddPersonnelFormRow` and `AddPersonnelFormRowProps` be better names for this component? As it's a row in a table